set.mm

```diff
@@ -22426,6 +22430,63 @@ primitive connectives (no defined terms) on the right-hand side.
       FBTUDCAUCJKLUDBCMNOABCPRS $.
   $}
 
+  $( Absorption of existence condition by uniqueness.  (Contributed by NM,
+     4-Nov-2002.)  Shorten proof and avoid ~ df-eu .  (Revised by BJ,
+     14-Oct-2022.) $)
+  moabs $p |- ( E* x ph <-> ( E. x ph -> E* x ph ) ) $=
+    ( wmo wex wi ax-1 wn nexmo pm2.6 ax-mp impbii ) ABCZABDZLEZLMFMGLENLEABHMLI
+    JK $.
+
+  $( Any proposition holds for some ` x ` or holds for at most one ` x ` .
+     (Contributed by NM, 8-Mar-1995.)  Shorten proof and avoid ~ df-eu .
+     (Revised by BJ, 14-Oct-2022.) $)
+  exmo $p |- ( E. x ph \/ E* x ph ) $=
+    ( wex wmo wo wn wi nexmo df-or mpbir ) ABCZABDZEKFLGABHKLIJ $.
```

Comment:
suggest to prove this using orri

Reply:
Thanks ! I apparently forgot to run `MM-PA> minimize_with *`